src/main/java/de/upb/crypto/math/structures/groups/elliptic/nopairing/Secp256k1.java

```diff
@@ -159,4 +168,46 @@ public boolean equals(Object obj) {
     public int hashCode() {
         return 0;
     }
+
+    public static class HashIntoSecp256k1 implements HashIntoGroupImpl {
+        private HashIntoZp hash;
+
+        /**
+         * Instantiate this hash function
+         * @param hash a random oracle hash function to the curve's base field
```

Comment:
supposedly random oracle, or what?

Reply:
What do you want me to do with that?

Reply:
Question does anyone using this method knows what to input if one reads "a random oracle hash function to the curve's base field"

Reply:
Addressed in 678b3fa